ibm/service/power/resource_ibm_pi_network_interface.go

```diff
@@ -0,0 +1,356 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+)
+
+func ResourceIBMPINetworkInterface() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPINetworkInterfaceCreate,
+		ReadContext:   resourceIBMPINetworkInterfaceRead,
+		UpdateContext: resourceIBMPINetworkInterfaceUpdate,
+		DeleteContext: resourceIBMPINetworkInterfaceDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
```

Comment:
Looks like Create time=Create+Update for this type of resource.

Reply:
Updated the create to only have create time